### ISSUE TITLE
64-bit DLL? 

### ISSUE THREAD

**Issue by tedsmith** [r1me/TTesseractOCR4#17]
In my endeavors to resolve issue https://github.com/r1me/TTesseractOCR4/issues/16 I am thinking part of my problem might be the expectation of 64-bit compilation for the API I am using to interract with TesseractOCR4. But to use the 64-bit edition, I need 64-bit DLL's of this library. 

Do you know where I might find those? Compiling them from source on WIndows looks painfully tedious, with several other tools requring download and installation etc.

**Comment by r1me** [comment]
https://digi.bib.uni-mannheim.de/tesseract/

**Comment by tedsmith** [comment]
Hi and thanks for the link above. 

My problem is that I am using this library to create a plugin that utilises a commercial API for a mainstream tool. That tool does come in both 32 and 64-bit versions, but, most users use the 64-bit version and as such my "plugin" needs to come in both a 32-bit but especially a 64-bit version. 

For 32-bit, everything is working well with what is provided here. But, for 64-bit, I have first installed the 64-bit version of the [MS Redistributable package](https://aka.ms/vs/16/release/vc_redist.x64.exe). I have then gone and compiled 64-bit DLL's of BOTH the Leptonica DLL and TesseractOCR DLL using MSYS2. Specifically, these two : 

https://packages.msys2.org/package/mingw-w64-x86_64-leptonica?repo=mingw64
https://packages.msys2.org/package/mingw-w64-x86_64-tesseract-ocr?repo=mingw64 

by triggering the following pacman commands : 

pacman -S mingw-w64-x86_64-tesseract-ocr 
pacman -S mingw-w64-x86_64-leptonica

These seem to have executed OK and I have two DLL's that I have copied to the bin subfolder and I have replaced the names of these DLL's in the tesseractocr.consts file, as follows : 

```
libleptonica = {$IFDEF Linux}'libpvt.cppan.demo.danbloomberg.leptonica-1.76.0.so'{$ELSE}'LibLeptonica-5-x64.dll';{$ENDIF};
libtesseract = {$IFDEF Linux}'libpvt.cppan.demo.google.tesseract.libtesseract-master.so'{$ELSE}'LibTesseractOCR-4-1-x64.dll';{$ENDIF};
``` 
I have used the 64-bit version of Lazarus, using "Default" in the target sections of the project (as that will create a 64-bit exe or DLL when running on 64-bit Windows) to compile my "plugin" for the commercial tool. All compiles OK. But when I run it with the debugger, it fails here: 

if (hTesseractLib = 0) then
    raise Exception.Create('Tesseract library is not loaded');     

Is there anything anyone can suggest? Or is getting a 64-bit DLL of BOTH libraries too tricky? 

Many thanks
